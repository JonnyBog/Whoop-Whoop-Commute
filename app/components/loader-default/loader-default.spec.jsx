import React from 'react';
import renderer from 'react-test-renderer';

import LoaderDefault from 'components/loader-default/loader-default';

describe('Components', () => {
  describe('LoaderDefault', () => {
    it('should export', () => {
      expect(LoaderDefault).toBeDefined();
    });

    /**
     * createRenderTree
     * @returns {JSX} component tree
     */
    function createRenderTree () {
      return renderer.create(
        <LoaderDefault />
      ).toJSON();
    }

    describe('Appearance', () => {
      it('should render', () => {
        const tree = createRenderTree();
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
