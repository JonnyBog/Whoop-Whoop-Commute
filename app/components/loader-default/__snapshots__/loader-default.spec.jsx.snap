// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components LoaderDefault Appearance should render 1`] = `
<div
  className="sc-jTzLTM rFSWX"
  color="#e8b52c"
  size="100"
/>
`;
